http/EffectiveUrlCache.h

```diff
@@ -83,7 +96,13 @@ class EffectiveUrlCache : public BESObj {
     std::shared_ptr<EffectiveUrl> get_effective_url(std::shared_ptr<url> source_url);
 
     void dump(std::ostream &strm) const override;
-    virtual std::string dump() const;
+
+private:
```

Comment:
A comment only - not a suggestion for a change because I have no idea how this would affect the BES code base, but this version of dump could be defined in the BESObj class once.

Reply:
We could try that and see how much angst it causes... ;-) (but not with this PR).